src/python/sdk/econia_sdk/utils/decimals.py

```diff
@@ -0,0 +1,106 @@
+import math
+from decimal import Decimal
+
+
+def get_lot_size_integer(smallest_decimal_unit: str, coin_decimals: int) -> int:
+    """
+    Given a decimal representation of the smallest possible unit and the
+    whole unit's decimals (e.g. ETH has 18 decimals), return the lot size.
+
+    Parameters:
+    * `smallest_decimal_unit`: Decimal (e.g. "0.001" for one thousandth) of
+      the smallest possible unit relative to a whole unit.
+    * `coin_decimals`: The number of decimals one whole unit of the coin
+      has (e.g. USDC has 6 decimals, ETH has 18).
+    """
+    smallest_decimal = Decimal(smallest_decimal_unit)
```

Comment:
To avoid potential float issues I suggest a type check on all `str` via a helper func like below:

```python
def verify_decimal_input(decimal_input: str):
    if type(decimal_input) is not str:
        raise ValueError("Decimal input should be a string")
```